### ISSUE TITLE
invalid scope error

### ISSUE THREAD

**Issue by rayjanoka** [spectrocloud/terraform-provider-spectrocloud#327]
I got this error when using `v0.14.3`, rolling back to `v0.14.2` worked.

```
Plan: 0 to add, 1 to change, 0 to destroy.
╷
│ Error: invalid scope
│
│   with spectrocloud_cluster_maas.self,
│   on kubernetes-spectro.tf line 243, in resource "spectrocloud_cluster_maas" "self":
│  243: resource "spectrocloud_cluster_maas" "self" {
│
╵
ERRO[0015] Terraform invocation failed in /tmp/terragrunt-cache/7G2UGMTyXMTczNDClSEqwZP-cRQ/z0pi8xjy06aSRPtb5mIKf9NGcEQ/kubernetes/spectro  prefix=[/Users/ray.janoka/code/terraform/dev]
ERRO[0015] 1 error occurred:
	* exit status 1

```

**Comment by nikchern** [comment]
Hello @rayjanoka, thank you for bringing this issue to our attention. To help us troubleshoot, could you provide additional details about how the "spectrocloud_cluster_maas" "self" resource is configured in your code? We are particularly interested in understanding the context or scope in which this resource was being created.

**Comment by rayjanoka** [comment]
hey, ya, it looks like this resource is linked to a `spectrocloud_cluster_profile` that is set to `context = "project"`.

**Comment by nikchern** [comment]
@rayjanoka issue should be fixed in 0.15.2 release with latest code changes. Please use that once it's out.

**Comment by nikchern** [comment]
fixed.

**Comment by nikchern** [comment]
thanks for letting me know @jhead-slg I filed ticket internally to check.

**Comment by nikchern** [comment]
@jhead-slg should be fixed in 0.16.1 version, however one should start configuration with a "clean" state file as in between there was incompatible attribute change.
